reporter/src/main/kotlin/utils/SpdxDocumentModelMapper.kt

```diff
@@ -194,6 +197,22 @@ private fun getSpdxCopyrightText(
 
 private fun Identifier.toSpdxPackageName(): String = "$type:$namespace:$name"
 
+private fun Package.toSpdxExternalReferences(): List<SpdxExternalReference> {
```

Comment:
Reading the signature, it looks odd to me that this function maps *one* `Package` to potentially *multiple* `SpdxExternalReferences`. Please change this to return `SpdxExternalReference` or `SpdxExternalReference?` and leave it to the caller to create a list.

Reply:
Ah, wait. Is the point that this might add e.g. Software Hertitage references in the future? Then I get the point of mapping *one* package to potentially *multiple* references.

Reply:
Yes, my id was to extend this function over time with Software Hertitage IDs and if possible CVE ids so one package may have multiple references to different systems.